### ISSUE TITLE
feat(docs): add self-hosted cli gettting started section

### ISSUE THREAD

**Issue by viceice** [renovatebot/renovate#5563]
**What would you like Renovate to be able to do?**
We should add some info from [docs/development/self-hosting.md](https://github.com/renovatebot/renovate/blob/master/docs/development/self-hosting.md)
to https://docs.renovatebot.com getting started section.

It would hopefully solve issues like renovatebot/config-help#548

**Comment by rarkins** [comment]
Yes, I think the self-hosting doc is mistakenly in `docs/development` for historical reasons. Probably the majority of it could be relocated to usage and then published to docs.

Documenting "administration" topics vs "usage" topics has always been a challenge. I don't want to confuse regular Renovate users with irrelevant bot admin topics if we can avoid it. Maybe the flow is "Installation" at top level and then "Self-Hosted" and "Hosted/App" and the latter is pretty minimal.
